.github/codecov.yaml

```diff
@@ -1,7 +0,0 @@
-coverage:
```

Comment:
We still get code coverage via their default settings, this just removes the status checks.